x11/xeyes/Portfile

```diff
@@ -12,9 +12,10 @@ long_description    xeyes defies description.
 homepage            https://www.x.org
 master_sites        xorg:individual/app/
 
-checksums           md5     a3035dcecdbdb89e864177c080924981 \
-                    sha1    efe6116d31a7f69e4fb6038613e52b0960b9b61c \
-                    rmd160  cc232409952e59e6d8e634c2728edde88bc4ed62
+checksums           size    133160 \
+                    sha1    458987f8accdbd8abf993ee51030391cf584d2e8 \
```

Comment:
I don't think sha1 is necessary at this point. Also, we traditionally put "size" last.

Reply:
dc5961f836cc268f288938ff2f32c825827613e0 done